plugin/src/shared/sc/plugin2021/Board.kt

```diff
@@ -14,6 +14,8 @@ class Board(
                 Array(Constants.BOARD_SIZE) { Array(Constants.BOARD_SIZE) { FieldContent.EMPTY } },
 ): IBoard {
     
+    constructor(toClone: Board) : this(Array(toClone.gameField.size) { toClone.gameField[it].clone() })
```

Comment:
`toClone` sounds a bit clumsy to me; I think it is obvious this is a copy ctor.
I'd go with something that integrates more into the read flow like `other`

Reply:
oh, ctor is just a shorthand for constructor, didn't realize that until now ^^ fair point